tests/test_browser.py

```diff
@@ -5103,6 +5103,17 @@ def test_assert_failure(self):
   def test_full_js_library_strict(self):
     self.btest_exit(test_file('hello_world.c'), args=['-sINCLUDE_FULL_LIBRARY', '-sSTRICT_JS'])
 
+  # Tests audio worklets
+  @requires_threads
+  @requires_sound_hardware
+  def test_audio_worklet(self):
+    self.btest(path_from_root('tests', 'audioworklet', 'audioworklet.cpp'),
+               expected='1',
```

Comment:
I wonder if you can use `btest_exit` here, and use `exit(0)` to signal success in your test (I'm trying to move browser tests in that direction in general).

Reply:
I don't think I can, because setting EXIT_RUNTIME will exit the runtime when `main()` returns, but I need to actually wait for the AudioContext to start, then for the worklet runs and do it's thing.